### PR TITLE
Prepare 3.1.2-SNAPSHOT

### PR DESCRIPTION
**Pull request by jmini**
Follow up from #588

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,8 +87,7 @@ OpenAPI Generator Version    | Release Date | OpenAPI Spec compatibility | Notes
 ---------------------------- | ------------ | -------------------------- | -----
 4.0.0 (upcoming major release) [SNAPSHOT](https://oss.sonatype.org/content/repositories/snapshots/org/openapitools/openapi-generator-cli/4.0.0-SNAPSHOT/)| TBD | 1.0, 1.1, 1.2, 2.0, 3.0 | Major release with breaking changes (no fallback)
 3.2.0 (upcoming minor release) [SNAPSHOT](https://oss.sonatype.org/content/repositories/snapshots/org/openapitools/openapi-generator-cli/3.2.0-SNAPSHOT/)| TBD | 1.0, 1.1, 1.2, 2.0, 3.0 | Minor release with breaking changes (with fallbacks)
-3.1.2-SNAPSHOT (current master, upcoming patch release) [SNAPSHOT](https://oss.sonatype.org/content/repositories/snapshots/org/openapitools/openapi-generator-cli/3.1.2-SNAPSHOT/)| TBD | 1.0, 1.1, 1.2, 2.0, 3.0 | Bugfix release
```
Comment (wing328) [inline README.md]: Shall we keep this line for 3.1.2-SNAPSHOT?

File: README.md [hunk 2]
```diff
@@ -145,16 +144,16 @@ See the different versions of the [openapi-generator-cli](https://mvnrepository.
 
 If you're looking for the latest stable version, you can grab it directly from Maven.org (Java 8 runtime at a minimum):
 
-JAR location: `http://central.maven.org/maven2/org/openapitools/openapi-generator-cli/3.1.1/openapi-generator-cli-3.1.1.jar`
+JAR location: `http://central.maven.org/maven2/org/openapitools/openapi-generator-cli/3.1.0/openapi-generator-cli-3.1.0.jar`
```
Comment (wing328) [inline README.md]: Looks like this change reverts 3.1.1 back to 3.1.0. Same for the other 2 line changes below.

File: modules/openapi-generator-gradle-plugin/README.adoc [hunk 3]
```diff
@@ -34,7 +34,7 @@ buildscript {
     mavenCentral()
   }
   dependencies {
-    classpath "org.openapitools:openapi-generator-gradle-plugin:3.1.1"
+    classpath "org.openapitools:openapi-generator-gradle-plugin:3.1.0"
```
Comment (wing328) [inline modules/openapi-generator-gradle-plugin/README.adoc]: 3.1.1 reverts back to 3.1.0

**Comment by jmini** [comment]
I have forced push my second commit that was containing to much changes (as pointed out by the review). It should be good now.

Thank you a lot for catching this!

**Pull request merged**